cns/restserver/restserver.go

```diff
@@ -171,6 +171,13 @@ func (service *HTTPRestService) Start(config *common.ServiceConfig) error {
 	listener.AddHandler(cns.V2Prefix+cns.AttachContainerToNetwork, service.attachNetworkContainerToNetwork)
 	listener.AddHandler(cns.V2Prefix+cns.DetachContainerFromNetwork, service.detachNetworkContainerFromNetwork)
 
+	createExtSwitchNetworkType, _ := service.GetOption(acn.OptCreateExtSwitchNetworkType).(string)
+	// Perform platform specific initialization
+	if err := platform.Init(createExtSwitchNetworkType); err != nil {
+		log.Printf("[Azure CNS] Failed with platform initialization due to error: %v", err)
+		return err
+	}
```

Comment:
This doesn't seem a right place to do the network creation. Main go seems like a right place to do it?

Should we be deleting it when CNS is stopped?

Reply:
Done